### PR TITLE
Filter feature

### PR DESCRIPTION
**Pull request by Rockbullar**
## Why

This pull request is needed because:

introduced filter functions to vestiaire
filter by price and size

[Link to Card]()

## What

This pull request introduces the following:

 * `Changes introduced`

`Add Implementation details if needed`

### Screenshot

`Attach any screenshots here. Remove when unneeded`

## How to Test

`Give the reviewer(s) some clues on how to test your PR`

## Other

`This section is for things that can't be put into the previous sections. Remove when unneeded`

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+    # @items = Item.all
+    # if params['categories']
+    #   categories = params[:categories]
+    #   @items = Item.where("categories ILIKE ?", "%#{categories}%")
+    # elsif params[:query].present?
+    #   condition = params[:query]
+    #   sql_query = "name ILIKE :query OR brand ILIKE :query OR categories ILIKE :query"
+    #   @items = Item.where(sql_query, query: "%#{condition}%")
+    # else
+    #   @items = Item.all
```
Comment (benedictng) [inline app/controllers/items_controller.rb]: remove commented code

**Comment by benedictng** [comment]
can set default min and max values?

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (Rockbullar) [reply app/controllers/items_controller.rb]: code removed!

**Comment by Rockbullar** [comment]
unable to set default values in a scope atm. Dev too junior. redirecting effort to other tasks

**Review by benedictng — APPROVED** [review]
lgtm

**Pull request merged**